### ISSUE TITLE
Fixture generation is not reproducible: no torch.manual_seed in export_fixtures.py

### ISSUE THREAD

**Issue by corey-cole** [corey-cole/executorch-numpy-runtime#16]
## Summary

`tools/export_fixtures.py` has no `torch.manual_seed`, so regenerating fixtures produces different
bytes every time for any model with learned parameters or a `randn` input. Anyone who regenerates
one fixture gets an unrelated diff on several others and has to hand-revert them.

Hit twice while adding `conv.pte` (#10) and `unplanned.pte` (#15); both times unrelated fixtures
came back byte-different and were reverted by hand.

## Which fixtures are affected

| Fixture | Nondeterministic? | Source of randomness |
|---|---|---|
| `conv.pte` | **yes** | `Conv2d` weight init + `torch.randn` input |
| `unplanned.pte` | **yes** | `Linear` weight init |
| `quantized.pte` | **yes** | `Linear` weight init + `torch.randn(2, 8)` calibration input |
| `add.pte` | no | no parameters, `torch.ones` inputs |
| `dtypes.pte` | no | no parameters |
| `multi.pte` | no | no parameters |
| `dynamic.pte` | no | no parameters |

The four deterministic ones are only deterministic by accident — they happen to have no learned
parameters. Any new fixture with a layer in it inherits the problem.

## Why it matters

- **Diff noise.** A PR touching one fixture arrives with two or three unrelated binary files
  changed. Reviewers cannot tell intentional from incidental, and the natural response — commit them
  all — silently rewrites fixtures other tests depend on.
- **Non-reproducible test inputs.** Two people regenerating the "same" fixture get different models.
  Any test that ever asserts on a value rather than a property becomes machine-dependent.
- **`quantized.pte` is the worst case.** Its calibration data is random, so quantization scales and
  zero-points differ per regeneration. That is exactly the kind of thing that produces a test which
  passes for one person and fails for another.

Upstream `executorch-runtime-dist` seeds explicitly for this reason — `scripts/emit-openvino-fixtures.py`
and `scripts/emit-xnnpack-fixtures.py` both carry `torch.manual_seed(0)` with the comment "the
fixture must be reproducible across releases for the same versions."

## Fix

Add `torch.manual_seed(0)` at the top of `main()` in `tools/export_fixtures.py`, with a comment
explaining why. One line.

**Cost: a one-time rewrite of `conv.pte`, `unplanned.pte`, and `quantized.pte`.** Their bytes change
once when the seed lands, and are stable from then on. That churn is the whole reason this was not
folded into #10 or #15 — it does not belong in a PR about something else.

## Verification

The fix is testable, and should be tested rather than assumed:

```bash
PATH="/path/to/et-venv/bin:$PATH" /path/to/et-venv/bin/python tools/export_fixtures.py tests/models
git status --short tests/models/          # expect: clean
```

Regenerating twice in a row must produce no diff. Worth a note in `tests/models/README.md` so the
property is stated rather than rediscovered.

## Out of scope

`tests/models/lstm/` ships its own fixture set from a different generator; if it has the same
problem it should be handled separately.
